ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -27,6 +27,8 @@ import SwiftUI
 ///
 /// `onTapAction`: perform an action on a cell tap.
 ///
+/// `hasDivider`:  displays a separator beneath the cell; default is set to `false`.
```

Comment:
nit : period

Reply:
Should I remove?

Reply:
I meant there is a semicolon inbetween your sentences.